internal/cmd/envs.go

```diff
@@ -57,6 +57,9 @@ func lsEnvsCommand() *cobra.Command {
 		Use:   "ls",
 		Short: "list all environments owned by the active user",
 		Long:  "List all Coder environments owned by the active user.",
+		PreRun: func(cmd *cobra.Command, args []string) {
+			autoStartInfo()
```

Comment:
I think this could get a little spammy for `envs ls`... maybe we write to a file and don't show the suggested > 1 time? or we only show it for `envs rebuild`?